test/pageStatus/test.js

```diff
@@ -81,18 +81,18 @@ define(['lodash', 'jquery', 'ui/pageStatus'], function(_, $, pageStatusFactory)
                 );
 
             _.delay(function() {
-                popup.close();
-            }, 200);
+                popup && popup.close();
+            }, 600);
 
             setTimeout(function() {
                 ready();
-            }, 500);
+            }, 900);
```

Comment:
This change is for some product decisions?

Reply:
window.open doesn't open popup immediately, I check locally, and change the delay

Reply:
:)